src/OrchardCore.Modules/OrchardCore.Redis/HealthChecks/RedisHealthCheck.cs

```diff
@@ -36,7 +36,7 @@ public async Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context
                 var time = await redisService.Database.PingAsync();
                 if (time > TimeSpan.FromSeconds(30))
                 {
-                    return HealthCheckResult.Unhealthy(description: "The Redis server couldn't be reached within {seconds} seconds and might be offline or have degraded performance.");
+                    return HealthCheckResult.Unhealthy(description: $"The Redis server couldn't be reached within {time.TotalSeconds} seconds and might be offline or have degraded performance.");
```

Comment:
That's not necessarily a correct message. If the response came in 31 seconds, then we'll reach this line, and the message will say "The Redis server couldn't be reached within 31 seconds", which would be untrue.

You need to put `30` into a const, and use it both for the timespan and the message.